### PR TITLE
feat: patient and facility endpoints on SDK

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#369

### Dependencies

- Upstream: none
- Downstream: https://github.com/metriport/metriport-internal/pull/437

### Description

BREAKING CHANGE:
- split facility create and update endpoints

Regular changes:
- add facility and patient mgmt functions
- add get facility endpoint
- fix patient contact schema
- expose facility, patient and USState schemas on sdk

### Release Plan

- nothing special, just merge along dependencies

### CODE REVIEW

File: api/app/src/routes/medical/facility.ts [hunk 1]
```diff
@@ -1,66 +1,99 @@
...
+
+    return res.status(status.OK).json(dtoFromModel(facility));
   })
 );
 
 /** ---------------------------------------------------------------------------
  * GET /facility
  *
- * Gets all of the facilities corresponding to the customer.
+ * Gets all of the facilities associated with this account.
```
Comment (Goncharo) [inline api/app/src/routes/medical/facility.ts:63]: 👍

File: packages/packages/api/src/medical/client/metriport.ts [hunk 2]
```diff
@@ -41,35 +50,145 @@ export class MetriportMedicalApi {
   /**
    * Creates an org or updates one if it already exists
    *
-   * @param {Organization}    organization    - The org you want to create or update
-   *
+   * @param organization The org you want to create or update
    * @returns The created or updated org.
    */
   async createOrUpdateOrganization(organization: Organization): Promise<Organization> {
```
Comment (Goncharo) [inline packages/packages/api/src/medical/client/metriport.ts:56]: Should we split the create/update for the org API while we're at it for consistency?

**Review by Goncharo — APPROVED** [review]
LGTM - one comment to consider about consistency here with the org API

**Review by Orta21 — APPROVED** [review]
Besides Dimas comments looks good

File: packages/packages/api/src/medical/client/metriport.ts (hunk 2 above)
Comment (leite08) [reply packages/packages/api/src/medical/client/metriport.ts:56]: I can, but I was hoping to merge this asap so I don't have to merge the suff from sandbox and can work on deploying the FHIR server.

Maybe add this as another task within the same ticket?

Comment (Goncharo) [reply packages/packages/api/src/medical/client/metriport.ts:56]: ^ sounds good to me

Comment (leite08) [reply packages/packages/api/src/medical/client/metriport.ts:56]: https://github.com/metriport/metriport-internal/issues/438

**Pull request merged**